src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -461,6 +528,150 @@ public void swipeLeft(int times) {
+     */
+    public Locomotive swipeSystemBack() {
+        Dimension screen = getAppiumDriver().manage().window().getSize();
+        Point start = new Point(screen.getWidth() - 2, getYCenter());
+        Point end = new Point(2, getYCenter());
+
+        return swipe(start, end);
+    }
+    //endregion Directional swipes
+
+    //region longPress swipes
+    public Locomotive longPressSwipeCenter(SwipeElementDirection direction) {
+        return longPressSwipe(direction, /*element=*/null, SWIPE_DISTANCE, 0);
+    }
+
+    public Locomotive longPressSwipeCenter(SwipeElementDirection direction, int swipeDurationInMillis) {
+        return longPressSwipe(direction, /*element=*/null, SWIPE_DISTANCE, swipeDurationInMillis);
+    }
+
+    public Locomotive longPressSwipeCenter(SwipeElementDirection direction, short numberOfSwipes) {
+        return longPressSwipeCenter(direction, 0, numberOfSwipes);
+    }
+
+    public Locomotive longPressSwipeCenter(SwipeElementDirection direction, int swipeDurationInMillis, short numberOfSwipes){
+        short i = 0;
+        while (i < numberOfSwipes) {
+            longPressSwipeCenter(direction, swipeDurationInMillis);
+            i++;
+        }
+        return this;
+    }
+
+    public Locomotive longPressSwipe(SwipeElementDirection direction, String id) {
+        return longPressSwipe(direction, PageUtil.buildBy(configuration, id));
+    }
+
+    public Locomotive longPressSwipe(SwipeElementDirection direction, By by) {
+        return longPressSwipe(direction, getAppiumDriver().findElement(by));
+    }
+
+    public Locomotive longPressSwipe(SwipeElementDirection direction, WebElement element) {
+        return longPressSwipe(direction, element, SWIPE_DISTANCE, 0);
+    }
+
+    public Locomotive longPressSwipe(SwipeElementDirection direction, String id, int swipeDurationInMillis) {
+        return longPressSwipe(direction, PageUtil.buildBy(configuration, id), swipeDurationInMillis);
+    }
+
+    public Locomotive longPressSwipe(SwipeElementDirection direction, By by, int swipeDurationInMillis) {
+        return longPressSwipe(direction, getAppiumDriver().findElement(by), swipeDurationInMillis);
+    }
+
+    public Locomotive longPressSwipe(SwipeElementDirection direction, WebElement element, int swipeDurationInMillis) {
+        return longPressSwipe(direction, element, SWIPE_DISTANCE, swipeDurationInMillis);
+    }
+
+    public Locomotive longPressSwipeCenterLong(SwipeElementDirection direction) {
+        return longPressSwipe(direction, /*element=*/null, SWIPE_DISTANCE_LONG, 0);
+    }
+
+    public Locomotive longPressSwipeCenterLong(SwipeElementDirection direction, int swipeDurationInMillis) {
+        return longPressSwipe(direction, /*element=*/null, SWIPE_DISTANCE_LONG, swipeDurationInMillis);
+    }
+
+    public Locomotive longPressSwipeCenterLong(SwipeElementDirection direction, short numberOfSwipes) {
```

Comment:
ditto about the overloads

Reply:
Good call. Removed the methods with a signature of only `short numberOfSwipes`.

Reply:
Good call. Removed the methods with a signature of only `short numberOfSwipes`.